frontend/src/utils/api.ts

```diff
@@ -237,13 +237,23 @@ window.api = {
       names.push(repoName);
     });
 
-    const blurbMap: { [key: string]: string } = data.blurbs.urlBlurbMap;
+    const repoBlurbMap: { [key: string]: string } = data.repoBlurbs.blurbMap;
+    const authorBlurbMap: {[key: string]: string} | undefined =
```

Comment:
let's simplify this line and the line below to:

```suggestion
    const authorBlurbMap: {[key: string]: string} | undefined = data.authorBlurbs?.blurbMap
```

Reply:
Alright! The code is updated!